core/main.go

```diff
@@ -10,10 +29,10 @@ import (
 	"os"
 	"time"
 
-	logger "github.com/edgexfoundry/go-mod-core-contracts/clients/logging"
-	"github.com/hashicorp/vault/api"
+	logger "github.com/edgexfoundry/edgex-go/pkg/clients/logging"
```

Comment:
this line needs to be what it was on master, i.e. change it to 
```go
logger "github.com/edgexfoundry/go-mod-core-contracts/clients/logging"
```
this package no longer exists in edgex-go, it needs to use the go-mod-core-contracts repo instead